stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -403,34 +403,42 @@ internal class ChannelControllerImpl(
 
     private fun loadMoreMessagesRequest(
         limit: Int = 30,
-        direction: Pagination
+        direction: Pagination,
+        messageId: String? = null,
     ): QueryChannelPaginationRequest {
+        return QueryChannelPaginationRequest(limit).apply {
+            val messageId = messageId ?: getLoadMoreBaseMessageId(direction)
+            messageId?.let {
+                messageFilterDirection = direction
+                messageFilterValue = it
+            }
```

Comment:
Is there a chance that `messageId` is `null` here? Does it make sense to make a request without these fields set?